bazel-mode.el

```diff
@@ -21,6 +21,31 @@
 ;;
 ;;; Code:
 
+(defcustom buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'string
+  :group 'bazel-mode)
```

Comment:
This should probably be `file`. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Simple-Types.html.

Reply:
Done. Thanks for the link.